src/examples/stm32/f4-disco/blinky4/blinky4.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+// Use goroutines to spin off led timing for each of the user LEDs on the stm32f4 discovery board
```

Comment:
the stm32f4 discovery has 4 user LEDs; I didn't want folks getting the idea that they could e.g run this as is on an Arduino Uno. The LEDs aren't a strong argument for it to be on its own, but the example I've got for SPI to the discovery board's might be as that calls the accelerometer that's not even on most other discovery boards.

I'm happy to move the LEDs example though, and will add them to the smoke tests as described.

Reply:
thinking about the SPI / accelerometer case, it's probably fine with the other examples too, as it just has general SPI <-> LIS302DL commands; this could work for any board / chip + the LIS302DL, so I think it can probably go with the rest of the examples.

Reply:
The main reason these examples are built in CI is to check that it compiles at all. Many trivial bugs are caught that way. So the specific example you're compiling doesn't matter all that much - as long as it basically works. As an example, the smoke test for the pca10056 board compiles examples/blinky1, even though it also has 4 LEDs on the board.
To be honest, I think a smoke test with just blinky1 or blinky2 would be fine, but I have no problem with adding another (small) example like blinky4. So I'm fine with the code as-is.

> I didn't want folks getting the idea that they could e.g run this as is on an Arduino Uno.

The same is also true for the blinky2 example. You'll immediately get a compiler error which should make the problem clear.